pkgs/development/libraries/intel-media-sdk/default.nix

```diff
@@ -11,12 +11,19 @@ stdenv.mkDerivation rec {
     sha256 = "1p13b4abslq31pbgqf0bzs2ixns85yfdsm94326h2vcg0q7hqc24";
   };
 
-  # patchelf is needed for binaries in $out/share/samples
-  nativeBuildInputs = [ autoPatchelfHook cmake pkgconfig ];
+  nativeBuildInputs = [ cmake pkgconfig gtest ];
```

Comment:
On the other hand, I'd put `gtest` in `checkInputs`.

Reply:
Done